### DISCUSSION TITLE
Questions about config file

### DISCUSSION THREAD

**Discussion by sunburn76** [swar/Swar-Chia-Plot-Manager#1131] [General]
Hello, I have some questions about how to edit the config file.

1. The ram (4000 by default): unit is mib or gb ?

2. Is it possible to only start first plotting with delay (for exemple the first 13?)

3. I saw everywhere that we need about 256 Gb SSD. But in case of parallel plotting, the "plotting temp files" are never 256 Gb at the same time, son do we really need 256 Gb ?
Here something I found :

![image](https://user-images.githubusercontent.com/85382096/120880390-89268200-c5ca-11eb-912d-7ac210ba340a.png)

4. I have Ryzen 3900x + 64 Gb + 2 x 2To Firecuda 520 raid 0.
Do I need an additional SSD (sata) temp ? (temporary2_directory)

5. Is it possible to open chia network (GUI) when plotting which Swar ?

Thank you

**Comment by realdevnullius** [comment]
Sigh... Ok then... :)

1. MB. Don't change it. If you have more RAM, the OS will use it as a cache memory. Who has 4TB RAM anyways??
2. dunno
3. yes, I even assume 3x the space in temp phases, but 2.5x should be enough (compared to final plotsize)
4. Don't use TMP2 for now, until you know how it goes without. Test data throughput of your HDDs volume and then compare it to what an SSD would give you (probably between 450 to 600MB)
5. Yes, it keeps your wallet synced and you can check the status of your plots.
